docs/content/en/docs/overview/_index.md

```diff
@@ -2,26 +2,28 @@
 title: "Overview"
 linkTitle: "Overview"
 weight: 10
-description: >
-  Provides an overview of EKS Anywhere
 ---
 
-EKS Anywhere creates a Kubernetes cluster on premises to a chosen provider.
-Supported providers include Bare Metal (via Tinkerbell), CloudStack, and vSphere.
-To manage that cluster, you can run cluster create and delete commands from an Ubuntu or Mac Administrative machine.
+### What is EKS Anywhere?
+EKS Anywhere is container management software built by AWS that makes it easier to run and manage Kubernetes clusters on-premises and at the edge. EKS Anywhere is built on [EKS Distro](https://distro.eks.amazonaws.com/), which is the same reliable and secure Kubernetes distribution used by [Amazon Elastic Kubernetes Service (EKS)](https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html) in AWS Cloud. EKS Anywhere simplifies Kubernetes cluster management through the automation of undifferentiated heavy lifting such as infrastructure setup and Kubernetes cluster lifecycle operations.
 
-Creating a cluster involves downloading EKS Anywhere tools to an Administrative machine, then running the `eksctl anywhere create cluster` command to deploy that cluster to the provider.
-A temporary bootstrap cluster runs on the Administrative machine to direct the target cluster creation.
-For a detailed description, see [Cluster creation workflow]({{< relref "../getting-started/overview/" >}}).
+Unlike Amazon EKS in AWS Cloud, EKS Anywhere is a user-managed product that runs on user-managed infrastructure. You are responsible for cluster lifecycle operations and maintenance of your EKS Anywhere clusters. EKS Anywhere is open source and free to use at no cost. To receive support for your EKS Anywhere clusters, you can optionally purchase [EKS Anywhere Enterprise Subscriptions]({{< relref "../concepts/support-scope/">}}) to get 24/7 support from AWS subject matter experts and access to [EKS Anywhere Curated Packages]({{< relref "../concepts/packages/">}}). EKS Anywhere Curated Packages are software packages that are built, tested, and supported by AWS and extend the core functionalities of Kubernetes on your EKS Anywhere clusters.
```

Comment:
```suggestion
Unlike Amazon EKS in AWS Cloud, EKS Anywhere is a user-managed product that runs on user-managed infrastructure. You are responsible for cluster lifecycle operations and maintenance of your EKS Anywhere clusters. EKS Anywhere is open source and free to use at no cost. To receive support for your EKS Anywhere clusters, you can optionally purchase [EKS Anywhere Enterprise Subscriptions]({{< relref "../concepts/support-scope/">}}). EKS Anywhere subscriptions get you:
* 24/7 support from AWS subject matter experts and
* Access to [EKS Anywhere Curated Packages]({{< relref "../concepts/packages/">}}). EKS Anywhere Curated Packages are software packages that are built, tested, and supported by AWS and extend the core functionalities of Kubernetes on your EKS Anywhere clusters.
```
I think it would help to get these two points to stick out a bit more, since that is what people are paying for.

Reply:
Going to keep it condensed, I purposefully don't want to be overselling here.